ai-platform-unified/notebooks/custom_job_image_classification_model_for_online_prediction.ipynb

```diff
@@ -1,2530 +1,2506 @@
 {
```

Comment:
<p>How about deleting the training job?</p><p> </p><p>https://github.com/googleapis/python-aiplatform/blob/master/samples/snippets/delete_custom_job_sample.py</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/248/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='120'/>

Reply:
I don't think this is blocking, since the `CustomJob` resource doesn't incur any further costs for the user after training is done. I'll file an issue about potentially add a section about deleting the job, which @morgandu or @andrewferlitsch can follow up on.

Reply:
Filed #250.